docs/tutorial/en/optimization/number_partition.ipynb

```diff
@@ -0,0 +1,260 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Conversion to PyQUBO by JijModeling transpiler\n",
+    "JijModeling has executed all the implementations so far.\n",
+    "By converting this to [PyQUBO](https://pyqubo.readthedocs.io/en/latest/), it is possible to perform combinatorial optimization calculations using OpenJij and other solvers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model,cache = to_pyqubo(problem,instance_data,{})\n",
+    "Q,offset = model.compile().to_qubo()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The PyQUBO model is created by `to_pyqubo` with the `problem` created by JijModeling and the `instance_data` we set to a value as arguments.\n",
+    "Next, we compile it into a QUBO model that can be computed by OpenJij or other software.\n",
+    "\n",
+    "### Optimization by OpenJij\n",
+    "\n",
+    "This time, we will use OpenJij's simulated annealing to solve the optimization problem.\n",
+    "We set the `SASampler` and input the `qubo` of the QUBO model into that sampler to get the result of the calculation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sampler = oj.SASampler(num_reads=100)\n",
+    "response = sampler.sample_qubo(Q=Q)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Decoding and displaying the solution\n",
+    "Let us take a look at the results obtained.\n",
+    "We decode the returned calculation results to facilitate analysis."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = cache.decode(response)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We sum the indices classified as $A_1$ and $A_0$ in $A$ here."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "class 1 : [ 1  2  3  4  6  7  9 10 11 12 14 18 23 24 28 29 30 32 33 37 38 39] , total value = 410\n",
+      "class 0 : [ 5  8 13 15 16 17 19 20 21 22 25 26 27 31 34 35 36 40] , total value = 410\n"
+     ]
+    }
+   ],
+   "source": [
+    "class_1_index = result.record.solution['x'][0][0][0]\n",
+    "class_0_index = [i for i in range(0,N) if i not in class_1_index]\n",
+    "\n",
+    "class_1 = instance_data['a'][class_1_index]\n",
+    "class_0 = instance_data['a'][class_0_index]\n",
+    "\n",
+    "print(f\"class 1 : {class_1} , total value = {np.sum(class_1)}\")\n",
+    "print(f\"class 0 : {class_0} , total value = {np.sum(class_0)}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As expected, both total values are 410."
```

Comment:
Above, we dealt with a problem whose solution is known because it is a continuous number.
We recommend you try more complex problems, such as generating numbers randomly.
を追加しました。

Reply:
continuous numberだと連続値、どちらかと言えば実数のように聞こえるので、consecutive numberの方が自然だと思います。